package/patterns-yast.spec

```diff
@@ -51,107 +50,124 @@ Provides:       pattern-visible()
 
 Requires:       libyui-ncurses-pkg
 Requires:       yast2
+Requires:       yast2-firewall
+Requires:       yast2-packager
+Requires:       yast2-theme
+
+Recommends:     yast2-metapackage-handler
+Recommends:     yast2-support
+
+Suggests:       yast2-firstboot
+Suggests:       yast2-snapper
+
+# SLE only
+%if !0%{?is_opensuse}
+Requires:       yast2-registration
+# yast2 clone_system is expected to be installed by default (sle-beta)
+Recommends:     autoyast2-installation
+%endif
+
 %if 0%{?is_opensuse}
-# opensuse only, see bsc#1125040
 Requires:       yast2-alternatives
+Recommends:     yast2-vm
+%endif
+
+# Don't install update facilities on Tumbleweed
+%if !0%{?suse_version} > 1500
```

Comment:
This would probably be better as `%if 0%{?sle_version}`, since Tumbleweed and Leap keep rolling up the version numbers.